components/Chat/Message/MessageContextMenu.tsx

```diff
@@ -101,14 +101,32 @@ const BackdropComponent: FC<{
     const getTransformValue = () => {
       if (itemRectY.value > AUXILIARY_VIEW_MIN_HEIGHT + safeAreaInsets.top) {
         const spacing = 16;
-        const topTransform =
+        const statusBarHeight =
+          Platform.OS === "android" ? StatusBar.currentHeight ?? 0 : 0;
```

Comment:
could be cool to centralize this logic inside a layout.ts file in theme or styles or constants or idk haha